test/libsolidity/InlineAssembly.cpp

```diff
@@ -291,27 +320,27 @@ BOOST_AUTO_TEST_CASE(oversize_string_literals)
 
 BOOST_AUTO_TEST_CASE(assignment_after_tag)
 {
-	BOOST_CHECK(successParse("{ let x := 1 { tag: =: x } }"));
+	BOOST_CHECK(successParse("{ let x := 1 { 7 tag: =: x } }"));
```

Comment:
No, I think my diagrams were wrong.  `x` is a let-declared variable, and is not a local variable.